ffi/capi/src/casemap.rs

```diff
@@ -354,6 +387,27 @@ pub mod ffi {
                 .titlecase_segment(s, &locale.0.id, options.into())
                 .write_to(write);
         }
+        /// Returns the full titlecase mapping of the given string, using compiled data (avoids having to allocate a TitlecaseMapper object)
+        ///
+        /// The `v1` refers to the version of the options struct, which may change as we add more options
```

Comment:
this should use an ABI rename instead of exposing this naming in bindings

Reply:
It needs to be exposed in bindings like C++ anyway.

This is the scheme we settled on after lots of discussion: we have suffixed APIs and structs, and on nonexhaustive backends like JS and Dart we unsuffix them.

Reply:
Either way, that's what we do here already.